docs/quality.md

```diff
@@ -1,3 +1,60 @@
 # Quality
 
-> @wip
+- Don't want to break this for driver, mongoose
+- Want input from these maintainers on establishing best practies, not to just
+  create more work
+- Result of this will be incorporated into our tooling,
+  [mj](https://github.com/rueckstiess/mj), for automated organization wide
+  checking and scaffolding for new projects
+
+## Testing
+
+- At the very least
+  - run mocha with one test that just tries to require your code
```

Comment:
I think we should avoid standardizing on a testing framework as f.ex the driver uses the homerolled one as I needed features specific to integration testing. Maybe we should just ensure we all use the correct npm hooks so that we can run tests etc.